sagemaker_batch_transform/custom_tensorflow_inference_script_csv_and_tfrecord/custom_tensorflow_inference_script_csv_and_tfrecord.ipynb

```diff
@@ -0,0 +1,717 @@
+{
```

Comment:
<p>Present tense and remove unnecessary comma:</p><p>"we'll use a very simple network architecture, with three densely-connected layers." -&gt; "we use a very simple network architecture with three densely-connected layers."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3438/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Done